regression-test/suites/delete_p0/test_zone_map_delete.groovy

```diff
@@ -54,4 +54,77 @@ suite("test_zone_map_delete") {
     qt_sql """select * from ${tableName} ORDER BY k1;"""
 
     sql """ DROP TABLE IF EXISTS ${tableName} """
+
+    // =========================
+    sql """
+    CREATE TABLE IF NOT EXISTS ${tableName} (
+        `k1` bigint(20) NULL,
+        `k2` largeint(40) NULL,
+        `k3` largeint(40) NULL
+    ) ENGINE=OLAP
+    AGGREGATE KEY(`k1`, `k2`, `k3`)
+    COMMENT 'OLAP'
+    DISTRIBUTED BY HASH(`k1`) BUCKETS 1
+    PROPERTIES (
+    "replication_allocation" = "tag.location.default: 1",
+    "in_memory" = "false",
+    "storage_format" = "V2",
+    "light_schema_change" = "true",
+    "disable_auto_compaction" = "false"
```

Comment:
I  think should disable auto compaction, then p0 could produce stable result. If disable auto compaction == false, the rowset maybe merged.